index.mdwn

```diff
@@ -34,28 +44,24 @@ window manager.
 [[!img images/6mon.small.png link="images/6mon.medium.png" alt="awesome running on 6 monitors"]]
 </div>
 
-# Features and non-features
-
-* Very stable, fast and small codebase and footprint.
-* First window manager using the asynchronous [XCB](http://xcb.freedesktop.org)
-  library instead of the old, synchronous
-  [Xlib](http://en.wikipedia.org/wiki/Xlib), which makes **awesome** less
-  subject to latency compared to other window managers.
-* Documented source code and API.
-* No mouse needed: everything can be performed with the keyboard.
-* Real multihead support (XRandR) with per screen desktops (tags).
-* Implements many [Freedesktop](http://www.freedesktop.org) standards:
-  [EWMH](http://standards.freedesktop.org/wm-spec/wm-spec-latest.html),
-  [XDG Base Directory](http://standards.freedesktop.org/basedir-spec/basedir-spec-latest.html),
-  [XEmbed](http://standards.freedesktop.org/xembed-spec/xembed-spec-latest.html),
-  [Desktop Notification](http://www.galago-project.org/specs/notification/),
-  [System Tray](http://standards.freedesktop.org/systemtray-spec/systemtray-spec-latest.html).
-* Does not distinguish between layers: there is no floating or tiled layer.
-* Uses tags instead of workspaces: allow to place clients on several tags, and
+# Awesome Key Features
+
+<div>
+<ul class="awesome">
+  <li>Very stable, fast and small codebase and footprint.
+  <li>Usage of the asynch <a href="http://xcb.freedesktop.org">XCB</a>
```

Comment:
i don't like this hidden surprises with unadvertised rewording

Reply:
> i don't like this hidden surprises with unadvertised rewording

yes. It is requirement from @sclu1034 too. He wants to keep original features point. I have to make short sentences to keep AW-points compact in small screens for vertical word awesomewm. But again, if the AW-points is removed, the original text will be used. 

Text content and correct english will be separate task.

About buttons. @sclu1034 wants link to actual released documentation. Again, I'm not sure with it. Because actual guides and a lot of staff was made in git documentation only. Do you really want links to 2 years old documentation? I know that it is a release, but.....you know....

Reply:
could you please send a comment on that particular line of code with a button, i'm not really following what exactly do you mean about it

Reply:
```
  <a href="[https://awesomewm.org/apidoc/documentation/07-my-first-awesome.md.html#](view-source:https://awesomewm.org/apidoc/documentation/07-my-first-awesome.md.html#)" title="Getting Started" class="btn btn-lg btn-primary mb-3 mb-lg-0 me-lg-3">
    Get Started
  </a>
  <a href="[https://awesomewm.org/apidoc/](view-source:https://awesomewm.org/apidoc/)" title="Documentation" class="btn btn-lg btn-outline-secondary mb-3 mb-lg-0 me-lg-3">
    Documentation
  </a>
```
**apidoc** is new documentation. It means for awesome-git. The released awesome and its documentation api is different and has another link. This is my main decision to focus to new apidoc, because you guides, one billion corrections in doc, better understanding etc. But we still haven't release of it. There is no easy decision and I don't want quick answer. My main change is mainly about these TWO BUTTONS, because I have minimally 2-3 questions every week to new apidoc and guides. It will spare my time very much, but not old documentation!

Reply:
mb leave buttons a links to stable doc, but below each of them put some sort of smaller text links which would lead to git-master doc?